Source/GSXib5KeyedUnarchiver.m

```diff
@@ -0,0 +1,3391 @@
+/** <title>GSXib5KeyedUnarchiver.m</title>
```

Comment:
I think things become critical when Github refuses to render a diff because it's too large. If this were at `$DAYJOB`, I'd refuse to review even this PR until it was split up, unless justification was given. Otherwise, even the sheer size of this single file is making it difficult to think straight about this file or keep my progress.

Given this file has a few smaller classes in it for decoding various elements in XIB5, how about we split this up into an approximation of one file per class? Plus, split up the absolutely huge `GSXib5KeyedUnarchiver` into... I don't know even what. But this is way too long to make sense of it.

Reply:
The smaller classes are just the ones you rightfully want to remove :-)
I think that the chance to remove these are higher if we leave the code here so it is visible.